identity-service/src/notifications/notificationQueue.js

```diff
@@ -66,7 +66,7 @@ async function _sendNotification (notifFn, bufferObj, logger) {
     logger.error(`${logPrefix} ERROR ${e.message}`)
   }
 
-  return numSentNotifs
+  return numSentNotifs || 0
```

Comment:
What is the value of `numSentNotifs` when it's not a number? just curious

Reply:
it is `NaN`, see PR description `processed NaN notifs`

Reply:
resolved offline, all good